src/arrayMethodSort.js

```diff
@@ -5,8 +5,37 @@
  */
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+    const compare = compareFunction || compareAsStrings;
+
+    for (let i = 1; i < this.length; i++) {
+      const unsortedElement = this[i];
```

Comment:
be more specific
```suggestion
      const firstUnsortedElement = this[i];
```

Reply:
I'll stick to my naming, there are no second or third unsortedElement down the line

Reply:
You got me on this one)))

Reply:
![image](https://user-images.githubusercontent.com/37161315/141316622-d2ab0908-328e-48d2-9349-920887447512.png)